moped-editor/src/views/projects/projectsListView/components/saveViewForm.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { useForm } from "react-hook-form";
+import FormControl from "@mui/material/FormControl";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { Button, Grid } from "@mui/material";
+import { CheckCircle } from "@mui/icons-material";
+
+import * as yup from "yup";
+import ControlledTextInput from "src/components/forms/ControlledTextInput";
+
+const validationSchema = yup.object().shape({
+  description: yup.string().required(),
+});
+
+const SaveViewForm = ({ onSave, description }) => {
+  const {
+    handleSubmit,
+    control,
+    watch,
+    formState: { errors },
+  } = useForm({
+    defaultValues: { description: description },
+    resolver: yupResolver(validationSchema),
+  });
+
+  const areFormErrors = Object.keys(errors).length > 0;
+
+  return (
+    <form onSubmit={handleSubmit(() => onSave())}>
```

Comment:
This anonymous function might be preventing the form data from making it's way to `onSave`. If you update to `handleSubmit(onSave)` then `formData` can be added as a parameter to `onSave`. 

You can see an example in `ProjectSummaryProjectDescription`.

Reply:
i don't know how i didn't notice this, thanks for catching it!